### ISSUE TITLE
How to send input stream through ssh connection

### ISSUE THREAD

**Issue by radimk** [int128/gradle-ssh-plugin#167]
I want to migrate code like this
`cat myBinaryFile | ssh ${SSH_OPTS} "some_command"`

Is this possible using interaction (https://gradle-ssh-plugin.github.io/docs/#_the_stream_interaction_support) even if there is nothing to match against?

**Comment by int128** [comment]
Currently it does not support sending input stream. Instead use a temporary file like:
```groovy
put from: 'myBinaryFile', into '/tmp'
execute 'some_command < /tmp/myBinaryFile'
```

Sending input stream will be supported in the future release.

**Comment by int128** [comment]
`standardInput` is available in the interaction closure since 2.0.0. Please try it.

```groovy
execute 'some_command', interaction: {
  new File('myBinaryFile').withInputStream { stream ->
    standardInput << stream
  }
}
```
